packages/shell-api/src/helpers.ts

```diff
@@ -327,6 +327,14 @@ export async function getPrintableShardStatus(
           autosplit === null || autosplit.enabled ? 'yes' : 'no',
       };
     })(),
+    (async (): Promise<void> => {
+      // Is automerge currently enabled, available since >= 7.0
+      const automerge = await settingsColl.findOne({ _id: 'automerge' });
```

Comment:
Not a problem to be solved in this PR, but the fact that we're doing 4 parallel `fineOne()`s on the same collection ... that's very how-our-users-use-MongoDB but not very how-users-should-use-MongoDB 😂

Reply:
truly committing to 'think like your user' 😄